llvm/test/CodeGen/X86/bf16-fast-isel.ll

```diff
@@ -0,0 +1,19 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 6
+; RUN: llc < %s -mtriple=x86_64-unknown-unknown | FileCheck %s
+
+define i8 @test(ptr %f) nounwind {
+; CHECK-LABEL: test:
+; CHECK:       # %bb.0: # %entry
+; CHECK-NEXT:    pushq %rax
+; CHECK-NEXT:    callq foo@PLT
+; CHECK-NEXT:    callq bar@PLT
+; CHECK-NEXT:    popq %rcx
+; CHECK-NEXT:    retq
+entry:
+  %call = call bfloat @foo(ptr %f)
+  %call2 = call noundef zeroext i8 @bar(bfloat %call)
+  ret i8 %call2
+}
+
```

Comment:
Can you add the same testcase wit Han indirect call. Can you also test each case with a non-default calling convention

Reply:
Added test case for indirect call and fast calling convention.